apps/daimo-mobile/src/view/screen/OnboardingScreen.tsx

```diff
@@ -478,7 +478,7 @@ function UseExistingPage({
           <View style={styles.vertQR}>
             <QRCode
               value={createAddDeviceString(pubKeyHex)}
-              color="#333"
+              color={color.grayDark}
```

Comment:
app no longer has hardcoded colors

named colors in the app match those on the website